external.bzl

```diff
@@ -617,6 +630,7 @@ def kythe_dependencies():
     _cc_dependencies()
     _go_dependencies()
     _java_dependencies()
+    _docker_dependencies()
```

Comment:
The tests are failing because our `bazel_skylib` version is incompatible with whatever the docker rules declare.  You can just move this in the after `_rule_dependencies()` below.